src/build.proj

```diff
@@ -13,7 +13,7 @@
     <WixBinSdkDirectory Condition="'$(WixBinSdkDirectory)' == ''">$(WixBinDirectory)sdk\</WixBinSdkDirectory>
 
     <Configuration Condition=" '$(Configuration)'=='' ">Release</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <Platform Condition=" '$(Platform)' == '' ">any cpu</Platform>
```

Comment:
Depends on JIT, though I don't see the reason to limit the output to x86. And we use "any cpu" on build server anyway.